examples/dht_node.rs

```diff
@@ -230,16 +237,16 @@ fn add_nat_sender(base_selector: IoFuture<()>, server_obj: &Server, pk: PublicKe
             err
         }))
 }
-fn add_nodes_sender(base_selector: IoFuture<()>, server_obj: &Server) -> IoFuture<()> {
+fn add_server_main_loop(base_selector: IoFuture<()>, server_obj: &Server) -> IoFuture<()> {
     // 20 seconds for NodesRequest
-    let interval = Duration::from_secs(20);
+    let interval = Duration::from_secs(1);
     let nodes_wakeups = Interval::new(Instant::now() + interval, interval);
     let server_obj_c = server_obj.clone();
     let nodes_sender = nodes_wakeups
         .map_err(|e| Error::new(ErrorKind::Other, format!("Nodes timer error: {:?}", e)))
         .for_each(move |_instant| {
             println!("nodes_wakeup");
-            server_obj_c.periodical_nodes_req()
+            server_obj_c.dht_main_loop(KILL_NODE_TIMEOUT, PING_TIMEOUT, PING_INTERVAL, BAD_NODE_TIMEOUT, NODES_REQ_INTERVAL)
```

Comment:
Better create DhtMainLoopArgs with named fields and pass only 1 arg struct instead of list of args